File: strategy_java/src/main/java/com/kmmania/strategy_training/DefaultTraining.java
package com.kmmania.strategy_training;

/**
 * Default implementation of the {@link AthleticTraining} interface.
 * This class represents the default training strategy.
 *
 * @author Dac Linh Giao NGUYEN
 * @version 0.1
 */
public class DefaultTraining implements AthleticTraining {

    /**
     * Method to perform the default training strategy.
     * This method prints a message indicating that default training is being performed.
     */
    @Override
    public void train() {
        System.out.println("*** Default training ***");
        System.out.println("...");
    }
}

File: strategy_kotlin/src/main/kotlin/strategy_training/DefaultTraining.kt
/*
 * Strategy project.
 * Dac Linh Giao NGUYEN
 * License MIT
 *
 * Strategy project in Kotlin.
 * DefaultTraining.kt
 */
package com.kmmania.strategy_training

/**
 * Default implementation of the [AthleticTraining] interface.
 * This class represents the default training strategy.
 *
 * @author Dac Linh Giao NGUYEN
 * @constructor Constructs a new instance of DefaultTraining.
 * @since 0.1
 * @see AthleticTraining
 */
class DefaultTraining : AthleticTraining {

    /**
     * Method to perform the default training strategy.
     * This method prints a message indicating that default training is being performed.
     */
    override fun train() {
        println("*** Default training ***")
        println("...")
    }
}